src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -25,12 +25,12 @@ public static IEnumerable<INamedTypeSymbol> GetBaseTypesAndThis(this INamedTypeS
             }
         }
 
-        public static Task<IEnumerable<INamedTypeSymbol>> FindDerivedClassesAsync(
+        public static Task<IEnumerable<INamedTypeSymbol>> FindTypesImmediatelyDerivedFromClassesAsync(
             this INamedTypeSymbol type,
             Solution solution,
             CancellationToken cancellationToken)
         {
-            return FindDerivedClassesAsync(type, solution, null, cancellationToken);
+            return DependentTypeFinder.GetTypesImmediatelyDerivedFromClassesAsync(type, solution, cancellationToken);
```

Comment:
Nitpick: can we just get rid of extension methods like this that just forward to a static somewhere? It just makes the codebase harder to navigate.